service/src/main/java/org/broadinstitute/listener/config/SamInspectorProperties.java

```diff
@@ -0,0 +1,24 @@
+package org.broadinstitute.listener.config;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+public class SamInspectorProperties {
```

Comment:
This seems like a good place to use `record`, assuming that the setters aren't used and the fields are actually `final`.